cl_sii/libs/xml_utils.py

```diff
@@ -31,6 +31,7 @@
 import signxml.exceptions
 import xml.parsers.expat
 import xml.parsers.expat.errors
+from cl_sii import __version__
```

Comment:
This does not seem right. libs modules should not depend on to the source package (`cl_sii` in this case), but the other way around.